README.md

```diff
@@ -7,13 +7,19 @@
 
 ## Developer Notes
 
-### Log folders
+- node v6 is required
+- `secret\index.js` will need to be created locally (copy `secret\index-sample.js`).
 
-The following folders are required to exist with chmod `777`.
+### Log folders
 
+For production, the following folders are required to exist with chmod `777`.
 - $HOME/www/log/webtorrent.io/crash-reports
 - $HOME/www/log/webtorrent.io/telemetry
 
+For local development (not using `--production` flag), these folders will be created at
```

Comment:
The way to put the server into production mode is actually `NODE_ENV="production"`, like most node server apps, not with a `--production` flag. I'll update this.